Feature/login first time popup

lilMell1 opened the pull request.

lilMell1 committed:
feature: added popup component +  enhanced login page

lilMell1 committed:
modified main to latest version

lilMell1 committed:
Merge branch 'dev' into feature/login-first-time-popup

IdoMotavasel reviewed (commented):
check and fix if needed the error message in the login form

lilMell1 committed:
fix: error msg in form + css

lilMell1 committed:
pulled latest version
Merge branch 'feature/login-first-time-popup' of https://github.com/core1400/frontend into feature/login-first-time-popup
